config/software/jemalloc.rb

```diff
@@ -2,14 +2,25 @@
 default_version "4.2.1"
 
 # for td-agent
-version("2.2.5") { source :md5 => 'a5c4332705ed0e3fff1ac73cfe975640' }
-version("3.6.0") { source :md5 => 'e76665b63a8fddf4c9f26d2fa67afdf2' }
-version("4.2.1") { source :md5 => '094b0a7b8c77c464d0dc8f0643fd3901' }
+version("2.2.5") do
+  md5 = 'a5c4332705ed0e3fff1ac73cfe975640'
+  source :md5 => md5
+  source :url => "http://pkgs.fedoraproject.org/repo/pkgs/jemalloc/jemalloc-#{version}.tar.bz2/#{md5}/jemalloc-#{version}.tar.bz2"
```

Comment:
You don't need another `source` -- just add a comma after `md5` (sorry for missing it before):
```ruby
  source :md5 => md5,
         :url => 'http://...'
```

Reply:
PTAL